.circleci/config.yml

```diff
@@ -46,85 +43,105 @@ jobs:
             find . -type f -regex ".*/spring-cloud-*.*/target/*.*" -exec cp {} $HOME/artifacts/ \;
             find . -type f -regex ".*/target/.*-reports/.*" -exec cp {} $HOME/artifacts/junit/ \;
             bash <(curl -s https://codecov.io/bash)
-      - run:
-          name: Install Snap
-          command: |
-            sudo apt update
-            sudo apt install snapd
-      - kube-orb/install-kubectl
-      - run:
-          name: Run Kind Integration Tests
-          command: |
-            cd spring-cloud-kubernetes-integration-tests
-            ./run.sh
-      - run:
-          name: Launch Kubernetes with microk8s
-          command: |
-            sudo snap install microk8s --classic --channel=1.11/stable
+          - run:
+              name: Install Snap
+              command: |
+                sudo apt update
+                sudo apt install snapd
+          - kube-orb/install-kubectl
+          - run:
+              name: Run Kind Integration Tests
+              command: |
+                cd spring-cloud-kubernetes-integration-tests
+                ./run.sh
+          - run:
+              name: Launch Kubernetes with microk8s
+              command: |
+                sudo snap install microk8s --classic --channel=1.11/stable
 
-            # wait until a k8s node is ready
-            sleep 10
-            n=0
-            until [ $n -ge 10 ]
-            do
-              (/snap/bin/microk8s.kubectl get no | grep -z "Ready") && break
-              n=$[$n+1]
-              sleep 20
-            done
+                # wait until a k8s node is ready
+                sleep 10
+                n=0
+                until [ $n -ge 10 ]
+                do
+                  (/snap/bin/microk8s.kubectl get no | grep -z "Ready") && break
+                  n=$[$n+1]
+                  sleep 20
+                done
 
-            echo "Kubernetes cluster launched"
+                echo "Kubernetes cluster launched"
 
-            # Allow intra-pod communication
-            sudo iptables -P FORWARD ACCEPT
+                # Allow intra-pod communication
+                sudo iptables -P FORWARD ACCEPT
 
-            echo n |/snap/bin/microk8s.enable dns registry istio
+                echo n |/snap/bin/microk8s.enable dns registry istio
```

Comment:
this `istio` profile is where I currently struggle (working on it), we have a single integration test that works with istio and to move it to kind take a while. I am working on it, but since you made this PR anyway, I will ask. We want to do that, still, correct? that is : move istio test to kind also

Reply:
yeah i think so

Reply:
np at all! thx for the feedback